fastqheat/eubioit_client.py

```diff
@@ -0,0 +1,80 @@
+import typing as tp
+
+import requests
+
+
+class EuBioITClient:
```

Comment:
I am open to suggestions for a better name for the class :)

Reply:
The European Bioinformatics Institute has many subdivisions, and since we work with ENA, the class name **must** contain ENA in its name

Reply:
@asuragan13 yes, since the API is called '[ENA Portal API](https://www.ebi.ac.uk/ena/portal/api/)` I think we should call it ENAClient in this case. I'll fix it

Reply:
fixed